.github/workflows/reusable_workflow.yaml

```diff
@@ -34,33 +34,35 @@ on:
       QUAY_PASSWORD:
         required: false
 
+env:
+  QUAY_TAG_ROOT: quay.io/${{ secrets.QUAY_NAMESPACE }}/${{ inputs.image_name }}
+  QUAY_IMG_EXP: 0
+  BUILD_PLATFORMS: linux/amd64, linux/arm64
+
 jobs:
   act-build:
     name: Build ${{ github.ref_name }} from ${{ github.event_name }}
     runs-on: ubuntu-latest
     steps:
-      - name: Set image tag for release
+      - name: Set image tags for release
         if: github.event_name == 'release'
         run: |
-          echo "IMAGE_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+          echo "IMAGE_TAGS=${{ env.QUAY_TAG_ROOT }}:${GITHUB_REF#refs/*/} ${{ env.QUAY_TAG_ROOT }}:latest" >> $GITHUB_ENV
       
-      - name: Set image tag for main
+      - name: Set image tags for main
         if: github.ref == 'refs/heads/main' && github.event_name != 'release'
         run: |
-          echo "IMAGE_TAG=main_latest" >> $GITHUB_ENV
+          echo "IMAGE_TAGS=${{ env.QUAY_TAG_ROOT }}:main_latest" >> $GITHUB_ENV
 
-      - name: Set image tag and expiration for dev
+      - name: Set image tags and expiration for dev
         if: github.ref != 'refs/heads/main' && github.event_name != 'release'
         run: |
           export commit_hash=${{ github.sha }}
-          echo "IMAGE_TAG=${GITHUB_REF##*/}_${commit_hash:0:7}" >> $GITHUB_ENV
           echo "QUAY_IMG_EXP=90d" >> $GITHUB_ENV
+          echo "IMAGE_TAGS=${{ env.QUAY_TAG_ROOT }}:${GITHUB_REF##*/}_${commit_hash:0:7}" >> $GITHUB_ENV
```

Comment:
This might be better implemented as a single `step` with the conditionals in the `run`.  Alternatively, instead of using `steps`, we could set `IMAGE_TAGS` as a job `ENV` variable, but the syntax for that is a little cumbersome, given that there are more than two options and the values are long.